.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    timeout-minutes: 4
```

Comment:
Having a timeout is very important. For example, on my personal account yesterday, while I was testing a GitHub actions setup, I had a test suite hang. That used up all of my free tier minutes for my account which me caught completely unawares.

Reply:
yikes - good to know